### DISCUSSION TITLE
Newbie Questions

### DISCUSSION THREAD

**Discussion by Berkgasm** [nerdyrodent/VQGAN-CLIP#53] [Q&A]
Hello, can we collect all newbie questions under this discussion?

For an example:

- Output always overwrites the file itself, is there any possible way to save every iteration separately like "export50.jpg, export100.jpg" ?

**Comment by Berkgasm** [comment]
Also how can i use Gumbel database, it says "AttributeError: module 'taming.models.vqgan' has no attribute 'GumbelVQ'"

**Comment by pma3d** [comment]
HI there. If I want to start a new session, where do I do that from? thanks
